database/influxdb/influxdb_user.py

```diff
@@ -0,0 +1,220 @@
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: influxdb_user
+short_description: Manage InfluxDB users
+description:
+    - Manage InfluxDB users
+version_added: 2.3
+author: "Adam Hamsik (@haad)"
+requirements:
+    - "python >= 2.6"
+    - "influxdb >= 0.9"
+options:
+    hostname:
+        description:
+            - The hostname or IP address on which InfluxDB server is listening.
+        required: true
+    username:
+        description:
+            - Username that will be used to authenticate against InfluxDB server.
+        default: root
+        required: false
+    password:
+        description:
+            - Password that will be used to authenticate against InfluxDB server.
+        default: root
+        required: false
+    port:
+        description:
+            - The port on which InfluxDB server is listening.
+        default: 8086
+        required: false
+    database_name:
+        description:
+            - Database name to work with.
+        required: true
+    user_name:
+        description:
+            - Name of the user that will be created/destroyed.
+        required: true
+    user_pass:
+        description:
+            - User password to create.
+        required: true
+    user_admin:
+        description:
+            - Determine if created user will be admin or not.
+        required: false
+    state:
+        description:
+            - Determines if the user should be created or destroyed.
+        choices: ['present', 'absent']
+        default: present
+        required: false
+'''
+
+EXAMPLES = '''
+# Example influxdb_user command from Ansible Playbooks
+- name: Create user
+    influxdb_user:
+      database_name: influx
+      hostname: "{{ influxdb_ip_address }}"
+      user_name: "{{ influxdb_user_name }}"
+      user_pass: "{{ influxdb_user_pass }}"
+      user_admin: no
+      state: present
+
+- name: Destroy user
+    influxdb_user:
+      database_name: influx
+      hostname: "{{ influxdb_ip_address }}"
+      user_name: "{{ influxdb_user_name }}"
+      state: absent
+
+- name: Create admin user
+    influxdb_user:
+      database_name: influx
+      hostname: "{{ influxdb_ip_address }}"
+      username: "{{ influxdb_username }}"
+      password: "{{ influxdb_password }}"
+      user_name: "{{ influxdb_user_name }}"
+      user_pass: "{{ influxdb_user_pass }}"
+      user_admin: yes
+      state: present
+'''
+
+RETURN = '''
+#only defaults
+'''
+
+try:
+    from influxdb import InfluxDBClient
+    from influxdb import exceptions
+    HAS_INFLUXDB = True
+except ImportError:
+    HAS_INFLUXDB = False
+
+
+def influxdb_argument_spec():
+    return dict(
+        hostname=dict(required=True, type='str'),
+        port=dict(default=8086, type='int'),
+        database_name=dict(required=True, type='str', default=False),
```

Comment:
This might be unrelated and could be an issue with my development environment (I should be running off the latest from devel), but this line gives me the error:

   ```
{
    "failed": true,
    "msg": "Module alias error: internal error: required and default are mutually exclusive for database_name"
}
```

Removing `default=False` fixes it. This might just be me, though.